src/scrapers/visa-cal.js

```diff
@@ -67,6 +69,7 @@ function convertTransactionType(txnType) {
     case WITHDRAWAL_TYPE_CODE:
     case WITHDRAWAL_TYPE_CODE_2:
     case REFUND_TYPE_CODE_2:
+    case NORMAL_TYPE_CODE_3:
```

Comment:
@eshaham since there is only one type representing installments and all the others represent normal transactions, we can avoid missing types support in the future by settings normal as default. Let me know if you want me to do it here, if so it will close #121 as well

Reply:
@esakal I actually like this process of approving txn types one by one, as it makes me think about each one, see my comment to the PR.